SpatialGDK/Documentation/setup-and-installing.md

```diff
@@ -151,4 +149,4 @@ Unreal provides a system called [Actor replication](https://docs.unrealengine.co
 To set up Actor replication:
 
 1. Set up your Actor for replication (including property replication and RPCs) using the [native Unreal workflow](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors) (Unreal documentation).
-1. Generate type bindings for your Actor (see the documentation on the [Interop Code Generator]({{urlRoot}}/content/interop)). This allows the SpatialOS Unreal GDK to serialize Unreal's replication data to SpatialOS.
+1. Tag the Actors and Subobjects you want to replicate with the `SpatialType` class tag. Then generate `schema` for your project via the `Schema Generator`. This allows the SpatialOS Unreal GDK to serialize Unreal's replication data to SpatialOS.
```

Comment:
(or use the generate for all supported classes <link>)

Reply:
As this is an experimental feature, lets no mention it yet. We can update this when it becomes the default.